puppet/zulip/files/logrotate/zulip

```diff
@@ -1,8 +1,13 @@
-/var/log/zulip/server.log /var/log/zulip/workers.log /var/log/zulip/manage.log {
+/var/log/zulip/*.log {
 	missingok
 	rotate 10
 	size 1G
 	compress
 	delaycompress
 	notifempty
+        create 644 zulip zulip
+        sharedscripts
+        postrotate
+                supervisorctl signal SIGHUP all >/dev/null 2>&1
+        endscript
```

Comment:
I think we only need this (which effectively is a full server restart) if we're managing supervisor logs; Django's own logging handles this sort of situation correctly.